tests/test_parsig_api.py

```diff
@@ -131,10 +151,10 @@ def test_parse_multiple_instructions2(self):
         self.assertEqual(result, expected)
 
     def test_parse_multiple_instructions3(self):
-        sig = "take two tablets of benadryl every two days and then 1 tablet every week for 1 month and than 1 tablet every month"
```

Comment:
Changing an existent test is very bad practice. the fact that you added new logic does not mean that the previous logic should not be tested